dev-requirements-py3.txt

```diff
@@ -1,12 +1,12 @@
 flake8>=3.7.8
-demisto-sdk>=0.3.3
-demisto-py>=2.0.7
+demisto-sdk==0.3.3
+demisto-py==2.0.8
```

Comment:
not sure about these... @ronykoz what do you think?

Reply:
2.0.8 introduced updates regarding ssl certificate, it's not mandatory to increase the version but shouldn't cause any issues.

Reply:
We might want the SDK to be always latest

Reply:
we want the option to release a new SDK version but keep working on a stable version for a few more days. (wouldn't like to release a new sdk in middle of content release and break the build).

Reply:
every SDK version is stable!